scripts/generator/graphql_generator/csharp/SDK/CartLineItems.cs.erb

```diff
@@ -147,7 +155,7 @@ namespace <%= namespace %>.SDK {
         ///
         /// cart.LineItems.AddOrUpdate(variantId, 3);
         /// \endcode
-        public void AddOrUpdate(string variantId, long? quantity = null, Dictionary<string, string> customAttributes = null) {
+        public void AddOrUpdate(string variantId, decimal price, long? quantity = null, Dictionary<string, string> customAttributes = null) {
             CartLineItem input = Get(variantId);
 
             if (input != null) {
```

Comment:
I think once we fix the main issue, by working on the ProductVariant level, instead of separate id and price, this sort of becomes less of an issue, but yes, I think it would update it.